tsconfig.json

```diff
@@ -64,5 +64,5 @@
     "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
   },
   "include": ["**/*.ts"],
-  "exclude": ["node_modules", "build"]
+  "exclude": ["node_modules", "build", "test"]
```

Comment:
Why do we want to include all testfiles from this config? It feels like they should be covered by our typescript config.

Reply:
In this line, I am excluding the testfiles, such that command `tsc` does not build js files from these ts-testfiles files.

Reply:
These include and exclude statements are about the files that the tsc script will transcript to .js files. 
And since we don't need the tests rewritten as js, we can exclude it here.

But still, this tsconfig is valid for all tests. E.g. the truffle-assertion imports would not work if we don't have the `typeRoots` set correctly.

Reply:
> the truffle-assertion imports would not work if we don't have the typeRoots set correctly.

Hmmm, it should work without `typeRoots` since the type definition gets included because of the include glob `**/*.ts`. Note that this is not the case with files transpiled by `ts-node` (which is what is being used by `truffle test` right now) as it ignores the `files`, `includes` and `excludes` by default: https://github.com/TypeStrong/ts-node#help-my-types-are-missing

Reply:
If we don't use this `typeRoots`, the declarations of types/truffle-assertions is not considered and we are getting the following error.

```
test/BatchExchange.spec.ts:9:27 - error TS7016: Could not find a declaration file for module 'truffle-assertions'. '/Users/alexherrmann/gnosis/dex-contracts/node_modules/truffle-assertions/index.js' implicitly has an 'any' type.
  Try `npm install @types/truffle-assertions` if it exists or add a new declaration (.d.ts) file containing `declare module 'truffle-assertions';`

9 import truffleAssert from "truffle-assertions";
```

Reply:
I think this is because the tests are compiled with `ts-node` and not with `tsc`.

I think the type roots configuration should stay, just commenting that it should not be necessary if we weren’t using `ts-node`.